src/components/details-panel/_macro-options.md

```diff
@@ -0,0 +1,27 @@
+| Name   | Type           | Required | Description                                             |
+| ------ | -------------- | -------- | ------------------------------------------------------- |
+| id     | string         | true     | HTML `id` attribute for the details panel               |
+| groups | array`<group>` | true     | Settings for the array of [group items](#group)         |
+| open   | boolean        | false    | Forces the details panel to be open when the page loads |
+
+## group
+
+| Name       | Type               | Required | Description                                                                                                                                                                                                                                        |
+| ---------- | ------------------ | -------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| title      | string             | true     | Heading for the groupItem and also title of the the details panel when there is one item in groupItems. <br> Example: "Correction" (for one item),"Corrections"(for multiple items) "Corrections and Notices" (multiple items and multiple groups) |
```

Comment:
I think we discussed having a title and a plural title that we would switch between based on the number of items